src/xcode/ENA/ENASecurity/Sources/ENASecurity/TrustEvaluation.swift

```diff
@@ -0,0 +1,67 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum TrustEvaluationError: Error {
+    case CERT_CHAIN_EMTPY
+    case CERT_PIN_NO_JWK_FOR_KID
+    case CERT_PIN_MISMATCH
+}
+
+class TrustEvaluation {
+
+    func check(trust: SecTrust, against jwkSet: [Data]) -> Result<Void, TrustEvaluationError> {
+        // Extract leafCertificate: the leafCertificate shall be extracted from the certificateChain. This is typically the first certificate of the chain.
+        if let serverCertificate = SecTrustGetCertificateAtIndex(trust, SecTrustGetCertificateCount(trust) - 1),
```

Comment:
The comment says "first certificate in the chain", but seems like the code takes the last?

Reply:
Good point. The comment is from the tech spec. 
I have copied the code from our certificate pinning.
Thats why I assume that the last one should be the leaf certificate in our case.
I think we will need to figure this out in a manual test.